### PR TITLE
Extract translations from modcontent chrome

### PR DESCRIPTION
**Pull request by PunkPun**

### CODE REVIEW

**Comment by PunkPun** [comment]
Linting is missing in modcontent

**Review by pchote — DISMISSED** [review]

File: OpenRA.Game/FileSystem/FileSystem.cs [hunk 1]
```diff
@@ -184,7 +184,7 @@ public void UnmountAll()
 		public void TrimExcess()
 		{
 			mountedPackages.TrimExcess();
-			explicitMounts.TrimExcess();
+			explicitMounts?.TrimExcess();
```
Comment (RoosterDragon) [inline OpenRA.Game/FileSystem/FileSystem.cs]: `explicitMounts` is never null.

**Review by RoosterDragon — APPROVED** [review]

File: OpenRA.Game/FileSystem/FileSystem.cs (hunk 1 above)
Comment (PunkPun) [reply OpenRA.Game/FileSystem/FileSystem.cs]: it was null when I was running the extraction rule for mode content. See the commit `Fix explicit mounts not being optional`

Comment (RoosterDragon) [reply OpenRA.Game/FileSystem/FileSystem.cs]: `explicitMounts` is never null by construction - are you sure it was this field? Either that or there's some reflection going on that I cannot trace.

Comment (PunkPun) [reply OpenRA.Game/FileSystem/FileSystem.cs]: interesting, it no longer crashes

**Review by RoosterDragon — APPROVED** [review]

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/51e90acd98483f86bfea86975c6dd8c6d64aa894

**Pull request merged**